api/MediaStreamAudioSourceNode.json

```diff
@@ -110,10 +110,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaStreamAudioSourceNode/mediaStream",
           "support": {
             "chrome": {
-              "version_added": "14"
+              "version_added": "23"
             },
             "chrome_android": {
-              "version_added": "18"
+              "version_added": "5"
```

Comment:
Set this to true.

Reply:
OK, done. Sorry about this. Clearly I'm not entirely back from the holiday break yet. :)